vllm/lora/models.py

```diff
@@ -238,8 +252,8 @@ def from_local_checkpoint(
                         f"While loading {lora_dir}, expected"
                         f" target modules in {expected_lora_modules}"
                         f" but received {unexpected_modules}."
-                        f" Please verify that the loaded LoRA module is correct"
-                    )
+                        f" Please verify that the loaded LoRA module "
+                        f"is correct")
```

Comment:
Not sure why this was changed
```suggestion
                        f" Please verify that the loaded LoRA module is correct"
                    )
```

Reply:
Not sure either.